libs/media-player/src/lib/use-key-down.ts

```diff
@@ -0,0 +1,34 @@
+import { useCallback, useEffect } from 'react';
+
+export enum KeyboardKey {
```

Comment:
This `enum` is serving 2 purposes.
1. Provides a useful handle on keyboard keys
2.  Defines the mapping from key to intent
I want you to decouple these because we already have plans to make the key bindings configurable.